packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,141 @@ enum SystemUiOverlay {
   bottom,
 }
 
+/// Describes the contrast needs of a color.
+enum Brightness {
+  /// The color is dark and will require a light text color to achieve readable
+  /// contrast.
+  ///
+  /// For example, the color might be dark grey, requiring white text.
+  dark,
+
+  /// The color is light and will require a dark text color to achieve readable
+  /// contrast.
+  ///
+  /// For example, the color might be bright white, requiring black text.
+  light,
+}
+
 /// Specifies a preference for the style of the system overlays.
 ///
 /// Used by [SystemChrome.setSystemUIOverlayStyle].
-enum SystemUiOverlayStyle {
+class SystemUiOverlayStyle {
   /// System overlays should be drawn with a light color. Intended for
   /// applications with a dark background.
-  light,
+  static const SystemUiOverlayStyle light = const SystemUiOverlayStyle(
+    systemNavigationBarColor: const Color(0xFFFFFFFF),
+    systemNavigationBarDividerColor: null,
```

Comment:
Don't we want a shade of gray here?

Reply:
Since I haven't gotten a chance to play with the P preview I don't really know what the behavior is.  null makes sure the API isn't invoked yet.

Reply:
Good point. I imagine that we'll have to render this divider on the framework side anyway.